src/entity/Living.php

```diff
@@ -465,7 +477,7 @@ public function damageArmor(float $damage) : void{
 		$this->armorInventory->setContents($armor);
 	}
 
-	private function damageItem(Durable $item, int $durabilityRemoved) : void{
+	public function damageItem(Durable $item, int $durabilityRemoved) : void{
```

Comment:
This change is not necessary and is outside the scope of this PR.

Reply:
Sorry, copy-paste leftovers from #4460